tests/unit/sagemaker/monitor/test_clarify_model_monitor.py

```diff
@@ -131,8 +131,8 @@
         "StartTimeOffset": START_TIME_OFFSET,
         "EndTimeOffset": END_TIME_OFFSET,
         "FeaturesAttribute": FEATURES_ATTRIBUTE,
-        "InferenceAttribute": INFERENCE_ATTRIBUTE,
-        "ProbabilityAttribute": PROBABILITY_ATTRIBUTE,
+        "InferenceAttribute": str(INFERENCE_ATTRIBUTE),
```

Comment:
why not define it directly as string?

Reply:
If you mean `INFERENCE_ATTRIBUTE`: it is the value of the `predictor.label` configuration in [analysis_config.json](https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-configure-processing-jobs.html#clarify-processing-job-configure-analysis), or the `label` argument of [ModelPredictedLabelConfig](https://github.com/aws/sagemaker-python-sdk/blob/bc05a58421bb0b4dc0b6bedc7ceb29a472ae39ff/src/sagemaker/clarify.py#L226), by definition it could be int (index) or str (JSONPath). In this case we define it as int to validate the fix.